### PR TITLE
fix: nested elements getting ignored

### PR DESCRIPTION
**Pull request by alihassan143**
#179

### CODE REVIEW

**Comment by alihassan143** [comment]
**Expected**
Screenshot 2023-06-09 at 5 34 01 PM
**Rendered Result**
Screenshot 2023-06-09 at 5 41 21 PM

**Comment by alihassan143** [comment]
**After Update**
https://github.com/AppFlowy-IO/appflowy-editor/assets/71745182/c1f98f15-20c1-4d5d-ad03-e366b1296209

**Comment by alihassan143** [comment]
@LucasXu0 kindly review it

**Comment by alihassan143** [comment]
#179

File: lib/src/editor/block_component/heading_block_component/heading_block_component.dart [hunk 1]
```diff
@@ -22,10 +22,12 @@ Node headingNode({
   required int level,
   Delta? delta,
   Attributes? attributes,
+  Iterable<Node>? children,
```
Comment (LucasXu0) [inline lib/src/editor/block_component/heading_block_component/heading_block_component.dart:25]: the heading can't embed the child node.

File: lib/src/plugins/html/html_document_decoder.dart [hunk 2]
```diff
@@ -133,17 +136,23 @@ class DocumentHTMLDecoder extends Converter<String, Document> {
...
       delta: delta,
     );
   }
 
   Iterable<Node> _parseBlockQuoteElement(dom.Element element) {
-    return element.children
-        .map((child) => _parseListElement(child, type: QuoteBlockKeys.type))
-        .toList();
+    final (delta, nodes) = _parseDeltaElement(element);
+    return [
```
Comment (LucasXu0) [inline lib/src/plugins/html/html_document_decoder.dart:149]: use `quoteNode` function instead.

```dart
return [quoteNode(...)]
```

**Comment by alihassan143** [comment]
@LucasXu0

**Pull request merged**